lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -457,7 +457,7 @@
 import time
 from ansible.module_utils.docker_common import DockerBaseClass
 from ansible.module_utils.docker_common import AnsibleDockerClient
-from ansible.module_utils.docker_common import HAS_DOCKER_PY_3
+from module_utils.docker_common import docker_version
```

Comment:
Ah, sorry, I think it also needs `ansible.` (as all the others). I wrote that down from my head, and it looks like I forgot something :)

Also, why don't you combine the above lines to:
```
from ansible.module_utils.docker_common import (
    DockerBaseClass, AnsibleDockerClient, docker_version,
)
```

Reply:
(Combining them is a matter of style, so feel free to ignore that :) It just came to my mind when I saw the lines...)

Reply:
Right... this time I tested it before pushing :smile:
I simply didn't know that syntax, It's definitely better.

Reply:
Looks like CI does not like that syntax, it's throwing [this](https://app.shippable.com/github/ansible/ansible/runs/91417/1/console) error
`````
02:49 Run command: /usr/bin/python test/sanity/validate-modules/validate-modules --format json --arg-spec lib/ansible/modules/cloud/docker/docker_swarm_service.py --base-branch origin/devel
02:50 ERROR: Found 1 validate-modules issue(s) which need to be resolved:
02:50 ERROR: lib/ansible/modules/cloud/docker/docker_swarm_service.py:0:0: E321 Exception attempting to import module for argument_spec introspection, 'cannot import name 'docker_version'' (75%)

`````

I'll go back to the more extensive form

Reply:
The problem is unrelated to this change, it was already caused by the change before (using `docker_version` instead of `HAS_DOCKER_PY_3`). But that can be easily fixed by setting `docker_version` to `None` if docker-py cannot be imported (see my comment below).